miatoll.json

```diff
@@ -1,24 +1,24 @@
 {
-	"response": [
-		{
-			"maintainer": "K O B U X",
-			"oem": "Redmi",
-			"device": "Redmi 9s series (miatoll)",
-			"filename": "crDroidAndroid-11.0-20210119-miatoll-v7.2.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/miatoll/7.x/crDroidAndroid-11.0-20210119-miatoll-v7.2.zip/download",
-			"timestamp": 1611047548,
-			"md5": "5086f16477146225bdb7127c557b07e7",
-			"size": 1115959961,
-			"version": "7.2",
-			"buildtype": "ALPHA",
-			"forum": "",
-			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-R/",
-			"firmware": "",
-			"modem": "",
-			"bootloader": "",
-			"recovery": "https://pitchblackrecovery.com/joyeuse",
-			"paypal": "",
-			"telegram": "https://t.me/crdroid_miatoll"
-		}
-	]
-}
\ No newline at end of file
+  "response": [
+    {
+        "maintainer": "Jamie Ho (henloboi)",
+        "oem": "Xiaomi",
+        "device": "Redmi Note 9 Pro / 9 Pro Max / 9S / Poco M2 Pro",
```

Comment:
Leave old device name (else breaks layout of device view from device)

Reply:
Fixed.